src/Effects.js

```diff
@@ -867,31 +987,69 @@ module.exports = class Effects {
       }
     };
 
-    this.strobe_lights = {
-      strobe: [],
+    this.pizzas = {
+      pizza: [],
       draw: function () {
         p5.background('black');
-        p5.strokeWeight(5);
-        p5.stroke(255);
-        let strobe = {
-          lineH: 25,
-          lineW: 1,
-          lineX: 0,
-          lightH: 50,
-          lightD: 40,
-        };
-        this.strobe.push(strobe);
-        this.strobe.forEach(function (strobe) {
+        for (let i = 0; i < 1; i ++) {
+          let pizza = {
+            x: 200,
+            y: 200,
+            velocity: p5.createVector(0, 1).rotate(p5.random(0,360)),
+            size: 0.1,
+          };
+          this.pizza.push(pizza);
+        }
+        p5.noStroke();
+        this.pizza.forEach(function (pizza){
           p5.push();
-          for (var x = 50; x <= 400; x += 100) {
-            p5.fill(p5.random(255), p5.random(255), p5.random(255));
-            p5.ellipse(x, strobe.lightH, strobe.lightD, strobe.lightD);
-          }
-          for (var y = 50; y <= 400; y += 100) {
-            p5.rect(y, strobe.lineX, strobe.lineW, strobe.lineH);
+          p5.translate(pizza.x, pizza.y);
+          drawPizza(p5._renderer.drawingContext);
+          let speedMultiplier = p5.pow(pizza.size, 2) / 8;
+          pizza.x += pizza.velocity.x * speedMultiplier;
+          pizza.y += pizza.velocity.y * speedMultiplier;
```

Comment:
I'm pretty sure `speedMultiplier` is less than 1 for the first several frames, so we're actually slowing the pizza down?